test/runtests.jl

```diff
@@ -3,6 +3,11 @@
 # Use of this source code is governed by an MIT-style license that can be found
 # in the LICENSE.md file or at https://opensource.org/licenses/MIT.
 
+import Pkg
+if get(ENV, "CI", "false") == "true"
+    Pkg.pkg"add MathOptInterface#master"
+end
```

Comment:
Remove

Reply:
Or does it need a new MOI release?

Reply:
Yes it needs a new MOI release, due to this test failure a couple commits ago: https://github.com/jump-dev/MiniZinc.jl/actions/runs/5634516877/job/15264515760
```
test_louvain: Error During Test at /home/runner/work/MiniZinc.jl/MiniZinc.jl/test/examples/louvain.jl:43
  Test threw exception
  Expression: MOI.Utilities.eval_variables((vi->begin
                x_sol[vi.value]
            end), model, f) == 1410
  MethodError: no method matching eval_variables(::Main.TestMiniZinc.var"#6#12"{Vector{Int64}}, ::MathOptInterface.Utilities.CachingOptimizer{MiniZinc.Optimizer{Int64}, MathOptInterface.Utilities.UniversalFallback{MathOptInterface.Utilities.Model{Int64}}}, ::MathOptInterface.ScalarNonlinearFunction)
```

Reply:
@odow I see MOI 1.19 is released, so I just updated this. I'm a bit dependent on this PR getting merged soon.